server/src/module/student/grading.service.ts

```diff
@@ -134,19 +133,11 @@ export class GradingService {
      * @throws `BadRequestException` - If an error occurs during the setting process of _any_ student this exception is thrown.
      */
     async setOfMultipleStudents(dtos: Map<Student, GradingDTO>): Promise<void> {
-        const em = this.entityManager.fork({ clear: false });
-        await em.begin();
-
-        try {
-            for (const [student, dto] of dtos) {
-                const handIn = await this.getHandInFromDTO(dto);
-                await this.updateGradingOfStudent({ student, dto, em, handIn });
-            }
-            await em.commit();
-        } catch (e) {
-            await em.rollback();
-            throw new BadRequestException(e);
+        for (const [student, dto] of dtos) {
+            const handIn = await this.getHandInFromDTO(dto);
+            await this.updateGradingOfStudent({ student, dto, handIn });
         }
+        await this.entityManager.flush();
```

Comment:
Is this still needed if no transaction is used?

Reply:
required for [implicit transactions](https://github.com/Dudrie/Tutor-Management-System/pull/1016#discussion_r1002301854)